### PR TITLE
Fix broken Links in IA Spec

### PR DESCRIPTION
**Pull request by jmealing**
Fix for issue: https://github.com/canada-ca/systeme-conception/issues/146

EN: https://github.com/canada-ca/design-system/pull/250

Preview: https://deploy-preview-147--systeme-conception-canada-ca.netlify.app/architecture/specifications-contenu-architecture-information-canada.html

### CODE REVIEW

File: architecture/organiser-contenu.md [hunk 1]
```diff
@@ -274,3 +274,12 @@ title: "Organiser le contenu dans Canada.ca"
...
   </section>
+  <section>
+    <nav role="navigation" class="mrgn-bttm-lg">
+      <h2 class="wb-inv">Navigation de document</h2>
+      <ul class="pager">
+        <li class="previous"><a href="elements-obligatoires.html" rel="prev">Précédent<span class="wb-inv">: Éléments obligatoires</span></a></li>
+        <li class="next"><a href="modeles.html" rel="next">Suivant<span class="wb-inv">: Concevoir le contenu</span></a></li>
+      </ul>
+    </nav>
+  </section>
```
Comment (delisma) [inline architecture/organiser-contenu.md:285]: Should we add this section in the English page also?

File: architecture/utilisation-concept-canadaca.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ title: "Qui doit utiliser le sysème de conception de Canada.ca"
...
-    <p>On encourage toutes les institutions à utiliser les composants de la&nbsp;<a href="/fr/secretariat-conseil-tresor/services/communications-gouvernementales/boite-outils-experience-web.html">Boîte à outils de l'expérience Web</a>&nbsp;pour leurs produits internes et externes. Ces composants sont gratuits et disponibles de façon ouverte. Ils reflètent les résultats des tests de convivialité.  Ils facilitent la  conformité aux exigences en matière d'accessibilité et de conception réactive  pour une diversité d'appareils avec un large éventail de tailles d'écran.</p>
+    <p>On encourage toutes les institutions à utiliser les composants de la&nbsp;<a href="https://www.canada.ca/fr/secretariat-conseil-tresor/services/communications-gouvernementales/boite-outils-experience-web.html">Boîte à outils de l'expérience Web</a>&nbsp;pour leurs produits internes et externes. Ces composants sont gratuits et disponibles de façon ouverte. Ils reflètent les résultats des tests de convivialité.  Ils facilitent la  conformité aux exigences en matière d'accessibilité et de conception réactive  pour une diversité d'appareils avec un large éventail de tailles d'écran.</p>
```
Comment (delisma) [inline architecture/utilisation-concept-canadaca.md:34]: In English the link points to http://www.tbs-sct.gc.ca/hgw-cgf/oversight-surveillance/communications/ws-nw/wet-boew-eng.asp. The one you've linked, is this a new page?

**Review by delisma — CHANGES_REQUESTED** [review]

File: architecture/utilisation-concept-canadaca.md (hunk 2 above)
Comment (jmealing) [reply architecture/utilisation-concept-canadaca.md:34]: I'm not sure which link is correct. No links were provided to me. I just fixed the broken link paths with paths that would work.

Comment (jmealing) [reply architecture/utilisation-concept-canadaca.md:34]: @donohuech do you know which link it should be, please?

**Comment by donohuech** [comment]
@jmealing 

Ces institutions should point here: https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception/liste-institutions.html 

Boite à outils de l’expérience Web should point here: https://www.canada.ca/fr/secretariat-conseil-tresor/services/communications-gouvernementales/boite-outils-experience-web.html

File: architecture/organiser-contenu.md (hunk 1 above)
Comment (prycrane) [reply architecture/organiser-contenu.md]: No.  It should be removed here.

**Review by prycrane — APPROVED** [review]

**Review by prycrane — APPROVED** [review]

**Review by prycrane — APPROVED** [review]

**Review by prycrane — APPROVED** [review]

**Pull request merged**